corehq/messaging/tasks.py

```diff
@@ -89,15 +89,22 @@ def initiate_messaging_rule_run(domain, rule_id):
     transaction.on_commit(lambda: run_messaging_rule.delay(domain, rule_id))
 
 
+def paginated_case_ids(domain, case_type):
+    case_generator = paginate_query_across_partitioned_databases(
+        CommCareCaseSQL,
+        Q(domain=domain, type=case_type, deleted=False),
+        values=['case_id']
+    )
+    for case in case_generator:
+        # return value from case generator is ('pk', 'case_id')
```

Comment:
it's not literal `('pk', 'case_id')`, right? Do you think it would be more clear if the comment said `('<pk>', '<case_id>')`.